randomizer/Lists/CrownLocations.py

```diff
@@ -39,7 +39,11 @@ def __init__(self, *, map=0, name="", x=0, y=0, z=0, scale=1, region=0, logic=No
             z=1804,
             scale=0.35,
             region=Regions.JungleJapesMain,
-            logic=lambda l: (l.handstand and l.islanky) or l.CanMoonkick() or ((l.phasewalk or l.generalclips) and (l.istiny or l.isdiddy)),
+            logic=lambda l: (l.donkey and l.advanced_platforming)
```

Comment:
Remove the DK & Advanced Platforming thing here. I think what you're referring to is DK being able to make this jump from the tree. Whilst this does work on N64, I believe this requires lag to make. It might be possible on PJ64/Wii U, but it's tough enough where I think it might be a bit out of scope of advanced platforming